app.js

```diff
@@ -2,7 +2,8 @@ const express = require("express");
 const { getTopics, 
   getArticleByID, 
   getUsers, 
-  patchUpdateVote} = require('./controllers/controllers')
+  patchUpdateVote,
+  getArticles} = require('./controllers/controllers')
```

Comment:
I would separate out the different data sets to their own controller e.g topicsController, articlesController otherwise it may get rather large and confusing